src/main/java/racing/car/ui/ResultView.java

```diff
@@ -11,12 +13,23 @@ public void outputView(List<Car> cars) {
         StringBuilder sb = new StringBuilder();
 
         for (Car car : cars) {
+            sb.append(car.getName()).append(" : ");
             generateRepeatedLine(car.getPosition(), sb);
             System.out.println(sb.toString());
+            sb.setLength(0);
         }
+
         System.out.println();
     }
 
+    public void outputWinnerView(int max, List<Car> cars) {
+        String winners = cars.stream()
+                .filter(car -> car.isWin(max))
+                .map(Car::getName)
+                .collect(Collectors.joining(", "));
+        System.out.println(winners + "가 최종 우승했습니다.");
+    }
```

Comment:
>ResultView에서 outputWinnerView 메서드를 테스트하고 싶었는데, 단순한 출력 메서드라서 테스트하기가 어렵습니다. 이런 경우에는 테스트를 생략해도 괜찮을까요?

우승자를 구하는 로직과 출력 로직이 함께 있어서 테스트하기 어렵다고 생각합니다.
ResultView는 출력에 대한 책임만 부여해주시고 우승자를 구하는 로직은 별도로 수행하면 어떨까요?
우승자를 구하는 로직에 대한 테스트도 없으니 추가하면 좋을 것 같아요.
```suggestion
    public void outputWinnerView(List<String> winnners) {
        System.out.println(String.join(", ", winners) + "가 최종 우승했습니다.");
    }
```